src/components/TeacherViews/Onboarding/Onboarding.view.tsx

```diff
@@ -98,96 +66,48 @@ export const OnboardingView = () => {
         }
       >
         <Logo height={48} width={104} variant="with text" />
-        <OakHeading tag="h2" $font="heading-light-5">
-          Select your school
+        <OakHeading
+          $mb={"space-between-s"}
+          $mt={"space-between-m"}
+          tag="h2"
+          $font="heading-light-6"
+        >
+          Do you work in a school?
         </OakHeading>
+        <OakBox $pv={"inner-padding-xl"}>
+          <OakRadioGroup
+            onChange={(value) =>
+              setWorksInSchool(value.target.value === "yes" ? true : false)
+            }
+            $flexDirection={"column"}
+            name={"Do you work in a school?"}
+          >
+            <OakRadioButton id="option-1" label="Yes" value="yes" />
+            <OakRadioButton id="option-2" label="No" value="no" />
+          </OakRadioGroup>
+        </OakBox>
 
-        <ResourcePageSchoolPicker
-          hasError={formState.errors?.school !== undefined}
-          schoolPickerInputValue={schoolPickerInputValue}
-          setSchoolPickerInputValue={onSchoolPickerInputChange}
-          schools={schools}
-          label={"School"}
-          setSelectedSchool={setSelectedSchool}
-          required={true}
-          withHomeschool={false}
-        />
         <OakPrimaryButton
+          $mv={"space-between-m"}
           disabled={
-            formState.errors?.school !== undefined || !formState.isValid
+            formState.errors?.worksInSchool !== undefined || !formState.isValid
+          }
+          element={
+            formState.errors?.worksInSchool !== undefined || !formState.isValid
+              ? "button"
+              : "a"
+          }
+          href={
+            // this should be added to url.tsx
```

Comment:
👁️ 👁️

Reply:
the focus state on this when it's a link is a bit off  
<img width="175" alt="Screenshot 2024-08-13 at 11 54 29" src="https://github.com/user-attachments/assets/e61b4714-13c0-4513-b684-4f170b0d9a74">

Reply:
As i mentioned in the comments above, i'm going to add this in another PR once this is in, to separate the page changes. There is some weirdness around the styling in the button that needs to be resolved as well.

Reply:
ah cool I missed that 👍